great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -202,15 +202,14 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
     def _init_variables(self) -> DataContextVariables:
         raise NotImplementedError
 
-    @abstractmethod
     def _save_project_config(self) -> None:
         """
-        Each DataContext will define how its project_config will be saved.
+        Each DataContext will define how its project_config will be saved through its internal 'variables'.
             - FileDataContext : Filesystem.
             - CloudDataContext : Cloud endpoint
             - Ephemeral : not saved, and logging message outputted
         """
-        raise NotImplementedError
+        self.variables.save_config()
```

Comment:
Leverage `variables` abstraction and remove the need for abstract method.

Reply:
Will hiding this `variables.save_config()` call inside this private method on this ABC make this `.save_config` call harder to follow?

Reply:
perhaps but until we can get rid of the impl of this method on `BaseDataContext`, we need it to be its own private method.